### PR TITLE
MICROS-6858 Add default env vars to orchestration

### PR DESCRIPTION
**Pull request by halcyonCorsair**
PR with initial vars, and to confirm approach.

@ash2k the shape we previously discussed is over in https://github.com/atlassian/voyager/compare/epahl/envvars-shape, but does not work for the initial use thing I wanted to use it for (ASAP keyserver env vars) because we always want to provide them, however asap isn't always wired in as a dependency, so we don't get the shape...

### CODE REVIEW

File: pkg/orchestration/wiring/k8scompute/plugin.go [hunk 1]
```diff
@@ -443,6 +450,23 @@ func buildContainers(spec *Spec, envDefault []core_v1.EnvVar, envFrom []core_v1.
...
+			Name:  awsRegionKey,
+			Value: string(location.Region),
+		},
+		{
+			Name:  envTypeKey,
+			Value: string(location.EnvType),
+		},
+		{
+			Name:  serviceNameKey,
+			Value: string(name),
```
[the remark this answers is not in the document]
Comment (halcyonCorsair) [reply pkg/orchestration/wiring/k8scompute/plugin.go]: Yes, this is wrong. 😱  It should be service name, not resource name.

[the remark this answers is not in the document]
Comment (halcyonCorsair) [reply pkg/orchestration/wiring/k8scompute/plugin.go]: Fixed.

File: pkg/orchestration/wiring/k8scompute/plugin.go [hunk 2]
```diff
@@ -460,6 +467,23 @@ func buildContainers(spec *Spec, envDefault []core_v1.EnvVar, envFrom []core_v1.
 	return containers
 }
 
+func buildDefaultEnvVars(name voyager.ResourceName, context wiringplugin.StateContext) []core_v1.EnvVar {
```
Comment (ash2k) [inline pkg/orchestration/wiring/k8scompute/plugin.go]: `name` is not used and can be removed

Comment (halcyonCorsair) [reply pkg/orchestration/wiring/k8scompute/plugin.go]: Done.

**Pull request merged**